src/Command/ModelCommand.php

```diff
@@ -1021,17 +1078,42 @@ public function bakeTable(Table $model, array $data, Arguments $args, ConsoleIo
             'rulesChecker' => [],
             'behaviors' => [],
             'connection' => $this->connection,
+            'persistentTable' => $removePersistentTable ? false : $usePersistentTable,
         ];
 
         $renderer = new TemplateRenderer($this->theme);
         $renderer->set($data);
         $out = $renderer->generate('Bake.Model/table');
 
-        $path = $this->getPath($args);
         $filename = $path . 'Table' . DS . $name . 'Table.php';
         $io->out("\n" . sprintf('Baking table class for %s...', $name), 1, ConsoleIo::QUIET);
         $io->createFile($filename, $out, $args->getOption('force'));
 
+        if ($usePersistentTable && !$removePersistentTable) {
+            if (!$persistentClassExists) {
+                $persistentOut = $renderer->generate('Bake.Model/persistent_table');
+                $io->out("\n" . sprintf('Baking persistent table class for %s...', $name), 1, ConsoleIo::QUIET);
+                $io->createFile($persistentTableFilePath, $persistentOut);
+
+                if (file_exists($persistentTableFilePath)) {
+                    require_once $persistentTableFilePath;
+                }
+            } else {
+                $io->out(
+                    "\n" . sprintf('Skipping creating persistent table class for %s...', $name),
```

Comment:
Why are there newlines being concatenated here? You could set `$newlines` to `2` instead.

Reply:
I guess I did copy paste from lines before https://github.com/cakephp/bake/blob/b83bdd87cd4c0fb3a2e6b246ac147e3d6a8d56d8/src/Command/ModelCommand.php#L1089 But yes, you are right. Will change in coming days.. I just synced my fork because I need it for one of my projects.